### PR TITLE
[OP#41289] list all linked workpackages 

### PR DESCRIPTION
**Pull request by individual-it**
fetches and shows all existing links between file and work-package

tested with https://github.com/opf/openproject/pull/10052

fixes https://community.openproject.org/projects/nextcloud-integration/work_packages/41289

https://user-images.githubusercontent.com/2425577/158364893-21afed40-5991-41a7-a2c0-86772ee3b04d.mp4

### CODE REVIEW

File: src/components/tab/SearchInput.vue [hunk 1]
```diff
@@ -148,48 +143,15 @@ export default {
...
-				const userName = workPackage._links.assignee.title
-				const avatarUrl = generateUrl('/apps/integration_openproject/avatar?')
-					+ encodeURIComponent('userId')
-					+ '=' + userId
-					+ '&' + encodeURIComponent('userName')
-					+ '=' + userName
-				const statusColor = await this.getWorkPackageColorAttributes('/apps/integration_openproject/statuses/', statusId)
-				const typeColor = await this.getWorkPackageColorAttributes('/apps/integration_openproject/types/', typeId)
+			for (let workPackage of workPackages) {
+				workPackage = await workpackageHelper.getAdditionalMetaData(workPackage)
```
Comment (kiranparajuli589) [inline src/components/tab/SearchInput.vue:147]: now the `getAdditionalMetaData` throws an error, shouldn't that be handled here?

**Review by SwikritiT — COMMENTED** [review]
LGTM other than already mentioned

File: src/components/tab/SearchInput.vue (hunk 1 above)
Comment (individual-it) [reply src/components/tab/SearchInput.vue]: logging an error now, see [c205b92](https://github.com/nextcloud/integration_openproject/pull/71/commits/c205b92229ce2e42f0e85d6b538000b5bfcf1d6e)

File: tests/jest/views/ProjectsTab.spec.js [hunk 2]
```diff
@@ -19,7 +19,12 @@ describe('ProjectsTab.vue Test', () => {
 	beforeEach(() => {
 		// eslint-disable-next-line no-import-assign
 		initialState.loadState = jest.fn(() => 'https://openproject/oauth/')
-		wrapper = shallowMount(ProjectsTab, { localVue })
+		wrapper = shallowMount(ProjectsTab, {
+			localVue,
+			data: () => ({
+				fileInfo: {},
```
Comment (kiranparajuli589) [inline tests/jest/views/ProjectsTab.spec.js:25]: why start fileInfo with an empty object only for a test, the component starts it with a null value. Should we update that information in the Vue component too?

Comment (individual-it) [reply tests/jest/views/ProjectsTab.spec.js:25]: yes `fileInfo` should always be an object

Comment (individual-it) [reply tests/jest/views/ProjectsTab.spec.js:25]: fixed in [e2d20bc](https://github.com/nextcloud/integration_openproject/pull/71/commits/e2d20bc345e5d2820b5596074e6de09d4121d588)

Comment (kiranparajuli589) [reply tests/jest/views/ProjectsTab.spec.js:25]: now that you've fixed it in the actual component, you may not need to init it from the test :).

Comment (individual-it) [reply tests/jest/views/ProjectsTab.spec.js]: done

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**